package/upgrade/lib.sh

```diff
@@ -208,9 +207,7 @@ import_image_archives_from_repo() {
         continue
       fi
 
-      curl -sfL $archive_url | zstd -d -f --no-progress -o $archive_file
-      $CTR -n k8s.io image import $archive_file
-      rm -f $archive_file
+      curl -fL $archive_url | zstdcat | $CTR -n k8s.io images import --no-unpack -
```

Comment:
A question:

Normaly, for `harvester` or others, there may only be a few images which are changed when upgrading. 

I ever checked the upgrade log, even when 1 image from the tar ball is changed, all the images in this tar ball will be imported again, but not sure the `ctr` will skip those existing ones automatically.

will below option of `ctr image import` be helpful?

```
--base-name="": base image name for added images, when provided only images with this name prefix are imported
```